packages/mobile/src/fees/saga.ts

```diff
@@ -1,51 +1,60 @@
 import { getStableTokenContract } from '@celo/walletkit'
 import { call, put, select, spawn, takeLeading } from 'redux-saga/effects'
+import { showError } from 'src/alert/actions'
+import { ErrorMessages } from 'src/app/ErrorMessages'
+import { ALERT_BANNER_DURATION } from 'src/config'
 import { getReclaimEscrowFee } from 'src/escrow/saga'
 import { Actions, EstimateFeeAction, feeEstimated, FeeType } from 'src/fees/actions'
 import { getInvitationVerificationFee } from 'src/invite/saga'
 import { getSendFee } from 'src/send/saga'
 import { CeloDefaultRecipient } from 'src/send/Send'
 import Logger from 'src/utils/Logger'
+import { web3 } from 'src/web3/contracts'
 import { currentAccountSelector } from 'src/web3/selectors'
 
 const TAG = 'fees/saga'
 
+// TODO: skip fee update if it was calculated recently
 export function* estimateFeeSaga({ feeType }: EstimateFeeAction) {
-  Logger.debug(TAG + '@estimateFeeSaga', `updating for ${feeType}`)
-
-  // TODO: skip fee update if it was calculated recently
-  const account = yield select(currentAccountSelector)
-
-  let feeInWei
-
-  switch (feeType) {
-    case FeeType.INVITE:
-      feeInWei = yield call(getInvitationVerificationFee)
-      break
-    case FeeType.SEND:
-      // Just use default values here since it doesn't matter for fee estimation
-      feeInWei = yield call(getSendFee, account, getStableTokenContract, {
-        recipientAddress: CeloDefaultRecipient.address,
-        amount: '1',
-        comment: 'Coffee or Tea?',
-      })
-      break
-    case FeeType.EXCHANGE:
-      // TODO
-      break
-    case FeeType.RECLAIM_ESCROW:
-      // Just use default values here since it doesn't matter for fee estimation
-      feeInWei = yield call(
-        getReclaimEscrowFee,
-        CeloDefaultRecipient.address,
-        CeloDefaultRecipient.address
-      )
-      break
-  }
+  Logger.debug(`${TAG}/estimateFeeSaga`, `updating for ${feeType}`)
+
+  try {
+    const account = yield select(currentAccountSelector)
+
+    let feeInWei
+
+    switch (feeType) {
+      case FeeType.INVITE:
+        feeInWei = yield call(getInvitationVerificationFee)
+        break
+      case FeeType.SEND:
+        // Just use default values here since it doesn't matter for fee estimation
+        feeInWei = yield call(getSendFee, account, getStableTokenContract, {
+          recipientAddress: CeloDefaultRecipient.address,
+          amount: web3.utils.fromWei('1'),
+          comment: 'Coffee or Tea?',
```

Comment:
why aren't we using the actual comment and amount the user is meaning to send?

Reply:
Because this fee estimation is done before the user inputs in amount and/or comment

Reply:
Maybe we should include something that is the max permissible comment length here?